apps/festival/festival/src/app/dashboard/home/home.component.ts

```diff
@@ -32,6 +34,7 @@ import { Intercom } from 'ng-intercom';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class HomeComponent {
+  public selectedCountry: string;
```

Comment:
Add `?` if not initialising
```suggestion
  public selectedCountry?: string;
```